ui/packages/app/web/src/components/ProfileSelector.tsx

```diff
@@ -154,6 +162,8 @@ const ProfileSelector = ({
     (error === undefined || error == null) && response !== undefined && response != null
       ? response.labelValuesList
       : [];
+  console.log('🚀 ~ file: ProfileSelector.tsx ~ line 154 ~ profileNames', profileNames);
```

Comment:
Was this intended to stay, or just for debugging?

Reply:
I assume just for debugging, just removed!